spec/features/feedback_spec.rb

```diff
@@ -80,6 +80,8 @@
       end
 
       background do
+        application.authority.update_attribute(:write_to_councillors_enabled, true)
```

Comment:
Great sleuthing! :mag: 

My change was definitely unnecessary then :frowning: As you were!